cmd/gitops/beta/run/cmd.go

```diff
@@ -830,12 +830,39 @@ func runCommandWithoutSession(cmd *cobra.Command, args []string) error {
 						log.Successf("Reconciliation is done.")
 					}
 
+					portForwards := map[string]*watch.PortForwardShortcut{}
```

Comment:
It feels a little odd that the key type here is a string while the map only ever holds numbers. We can just as well make it a slice:
```suggestion
					portForwards := make([]watch.PortForwardShortcut, 0)
```

Reply:
good point, I changed the suggestion.